sips/sip-028/sip-028-sbtc_peg.md

```diff
@@ -0,0 +1,217 @@
+
+While up to 30% of the signers can be offline without a user impact on the functioning of the protocol, it becomes more critical for the rest of the signers to approve sBTC operations because operations necessarily still need to meet 70% of the original signing power. If more than 30% of signers become unavailable, no sBTC operations will be approved because it will be impossible to get 70% approval when less than 70% are online. An operation that isn’t approved will become spendable by the user after a predetermined timeout has elapsed, measured in Bitcoin blocks.
+
+### sBTC Signer Responsibilities
+
+Overview of tasks the sBTC signers carry out:
+    - They react to requests to deposit BTC.
+    - They react to requests to withdraw BTC.
+    - They stay online often enough that deposits and withdrawals happen in a timely manner.
+    - They have to keep their signer hosts secure, so their keys don't get stolen.
+    - They have to proactively re-key every so often, which means moving the BTC to a new UTXO.
+- Signers must consolidate BTC as it is deposited.
+    - The process for UTXO consolidation of the BTC peg wallet is described in this [issue](https://github.com/stacks-network/sbtc/issues/52).
+- Signers must deduct transaction fees from users in order to fund BTC withdrawal transactions.
+    - They must ensure that the transaction fee is paid for (e.g., they deduct it from the user, and they set a minimum sBTC withdrawal amount).
+    - The process for handling transaction fee estimates is described in this [issue](https://github.com/stacks-network/sbtc/pull/186).
+- Signers must coordinate to set and advertise the fee parameters of the system.
+    - They must decide a minimum sBTC peg-out.
+    - They must decide an STX transaction fee for minting the sBTC. This fee is paid by the user and can be sponsored by a 3rd party, which is described in the “Auxiliary Features” section of this document.
+
+### sBTC Signer Eligibility Criteria
+
+Signers will run the sBTC binary in addition to the core Stacks signer software and must meet certain criteria in order to facilitate the reliable functioning of the sBTC protocol at all times.
+
+The following eligibility criteria will be used to identify the sBTC Signers:
+
+- **Company Standing:**  
+  Does the company have an operating history to demonstrate their experience and reliability?
+- **Stacks Participation:**  
+  Has the Signer participated in running a signer node on Stacks 2.5 testnet or mainnet?
+- **Network Uptime:**  
+  Does the Signer agree to use reasonable efforts to maintain >99% uptime on the sBTC Signer?  
+  *Note: This metric will be self-affirmed by the company.*
+- **Communication & Availability:**  
+  Does the signer have a direct communication channel set up with sBTC core engineers to be able to respond to urgent updates within 24 hours? (e.g., Slack, Telegram, Signal)
```

Comment:
Is there a way for them to demonstrate their availability?  As in, is there a way for members of the public to confirm that they're present in these channels?

Also, why are there more than one channel?  Don't we want all the signers to be using the same channel, so there's a single well-known way to get ahold of them if something breaks?

Reply:
i beleive this is addressed with changes in 321d118f5dd48f1f526bd6d9cfd89497878ac98b